artifacts.go

```diff
@@ -150,52 +155,61 @@ func run() error {
 	writeRootfs.Flags = []cli.Flag{
 		cli.StringFlag{
 			Name:  "update, u",
-			Usage: "Update `FILE`",
+			Usage: "Update `FILE`.",
 		},
-		cli.StringFlag{
-			Name:  "device-type, t",
-			Usage: "Type of device supported by the update",
+		cli.StringSliceFlag{
+			Name: "device-type, t",
+			Usage: "Type of device(s) supported by the update. You can specify multiple " +
+				"compatible devices providing this parameter multiple times.",
 		},
 		cli.StringFlag{
 			Name:  "artifact-name, n",
 			Usage: "Name of the artifact",
 		},
 		cli.StringFlag{
 			Name:  "output-path, o",
-			Usage: "Full path to output artifact file",
+			Usage: "Full path to output artifact file.",
+		},
+		cli.IntFlag{
+			Name:  "version, v",
+			Usage: "Version of the artifact.",
+			Value: 1,
```

Comment:
I don't think this will be configurable, will it? Or are you anticipating one tool being able to write several distinct versions?